R/bind_ard.R

```diff
@@ -7,10 +7,18 @@
 #'   ARDs to combine. Each argument can either be an ARD,
 #'   or a list of ARDs. Columns are matched by name, and any missing
 #'   columns will be filled with `NA`.
+#' @param .distinct (`logical`)\cr
+#'   logical indicating whether to remove non-distinct values from the ARD.
+#'   Duplicates are checked across grouping variables, primary variables,
+#'   context (if present), the **statistic name and the statistic value**.
+#'   Default is `FALSE`. If a statistic name is repeated and `.update=TRUE`,
```

Comment:
```suggestion
#'   Default is `FALSE`. If a statistic name is repeated and `.update = TRUE`,
```
You can discard

Reply:
should it be `.distinct = TRUE` ?

Reply:
```suggestion
#'   Default is `FALSE`. If a statistic name and value is repeated and `.distinct = TRUE`,
```

Reply:
ahh, thank you for finding this 😬 UPDATED